### ISSUE TITLE
Feature request: add option to show crowdsourced title under original title

### ISSUE THREAD

**Issue by mariomadproductions** [ajayyy/DeArrow#232]
In smaller text and/or different style. This would mean you can see both the original and crowdsourced title at a glance, rather than requiring lots of mouse clicks.

**Comment by jgoedde** [comment]
Fyi, there is this option that, when actived, shows the original title as soon as you hover over the blue dot on a youtube video so you don't have to click.

![image](https://github.com/ajayyy/DeArrow/assets/129423545/827e65fe-42b6-4325-8d22-27d44967e19f)

Although I think something permanent like a small text below the title would be cool too.

**Comment by mariomadproductions** [comment]
Yeah this is the same sort of idea, just without requiring you to hover, which slows things down quite a bit.

**Comment by aspizu** [comment]
![](https://media.discordapp.net/attachments/1216842627379363924/1244388803624308856/image.png?ex=6654eee2&is=66539d62&hm=91a352aef8915e0194837643df01a18810a370d1cf5ee0c3d1797ef01ed11bf8&=&format=webp&quality=lossless&width=1100&height=196)

**Comment by mariomadproductions** [comment]
@aspizu is that a mockup? looks good. original title is the crossed out one i guess.

**Comment by aspizu** [comment]
Yes, it is just a mockup I made using devtools. The original title is crossed-out.
